Cargo.toml

```diff
@@ -135,6 +136,9 @@ features = ["nightly", "push", "process"]
 version = "0.1.4"
 default-features = false
 
+[patch."https://github.com/pingcap/kvproto.git"]
```

Comment:
Please make the patch under the "service-test" feature.

Reply:
You can't put patch under a feature